public/wp-content/themes/ColoredCow/functions.php

```diff
@@ -22,30 +22,27 @@ function cc_styles() {
 //add filter to remove margin above html
 add_filter('show_admin_bar','__return_false');
 
-function add_subscriber()
+
+function verify_field2()
```

Comment:
please rename the function. The functionality is not clear from the name. It looks like it validates field2 but what exactly is field2?